README.md

```diff
@@ -184,11 +186,11 @@ nuocollector:
         data_format = "influx"
 ```
 
-# Setup on Bare Metal
+# Setup on Bare Metal Linux
 
 ## Installation
 
-These steps are for RedHat or CentOS. For other platforms, see [Telegraf Documentation](https://portal.influxdata.com/downloads/).
+These steps are for Red Hat or CentOS bare-metal hosts or VMs. For other platforms, see [Telegraf Documentation](https://portal.data.com/downloads/).
```

Comment:
the change to the link is incorrect.

Reply:
Thanks, not even sure how I managed to change that, was only attempting to change the text portion! Fixed.